### DISCUSSION TITLE
Simplification of SUNF due to symmetry

### DISCUSSION THREAD

**Discussion by nmoynihan** [FeynCalc/feyncalc#150] [General]
I am trying to simplify expressions involving FieldStrength and SUNF[I,j,k]. If I have an expression like:

![image](https://user-images.githubusercontent.com/1060322/150185396-7b2c60f8-41ee-4b67-9562-00b538df3b85.png)

Is there a way I can simplify this further, i.e. recognise that the symmetries of SUNF will simplify this expression?

Code, in case it is useful:

SUNF[i,j,k](FieldStrength[\[Mu],\[Nu],i]FieldStrength[\[Nu],\[Rho],j]FieldStrength[\[Rho],\[Mu],k] - FieldStrength[\[Mu],\[Nu],j]FieldStrength[\[Nu],\[Rho],i]FieldStrength[\[Rho],\[Mu],k])//SUNSimplify

**Comment by vsht** [comment]
Only by hand

```
exp = SUNF[i, j, 
   k] (FieldStrength[\[Mu], \[Nu], i] FieldStrength[\[Nu], \[Rho], 
      j] FieldStrength[\[Rho], \[Mu], k] - 
    FieldStrength[\[Mu], \[Nu], j] FieldStrength[\[Nu], \[Rho], 
      i] FieldStrength[\[Rho], \[Mu], k])

FS[a_, b_, c_] /; ! OrderedQ[{a, b}] := -FS[b, a, c];

exp /. FieldStrength[a_, b_, c_] :> 
  FS[LorentzIndex[a], LorentzIndex[b], SUNIndex[c]]
% // FCCanonicalizeDummyIndices
```

**Reply by nmoynihan** [reply]
Thanks!
